src/Symfony/Bundle/FrameworkBundle/Templating/Helper/AssetsHelper.php

```diff
@@ -94,8 +94,13 @@ private function getLegacyAssetUrl($path, $packageName = null, $version = null)
     {
         if ($version) {
             $package = $this->packages->getPackage($packageName);
+            $class = new \ReflectionClass($package);
 
-            $v = new \ReflectionProperty($package, 'versionStrategy');
+            while ('Symfony\Component\Asset\Package' !== $class->getName()) {
```

Comment:
same comment than in the original PR: why can't we just do `$v = new \ReflectionProperty('Symfony\Component\Asset\Package', 'versionStrategy');` instead of the while loop?